src/test/java/com/squareup/kotlinpoet/TypeAliasSpecTest.kt

```diff
@@ -37,6 +37,18 @@ class TypeAliasSpecTest {
     }
   }
 
+  @Test fun typeVariable() {
+    val v = TypeVariableName("V")
+    val typeAliasSpec = TypeAliasSpec
+        .builder("Word", ParameterizedTypeName.get(List::class.asClassName(), v))
```

Comment:
It's a bit weird to reference the variable in the supertype first, but we can look at that later...